### DISCUSSION TITLE
Testing email templates in local development

### DISCUSSION THREAD

**Discussion by rienheuver** [supabase/supabase#14738] [Feature Requests]
Right now I can't test my email templates before putting them on production. It would be useful to test mail template to check if they are working and looking as expected before saving them on production.
Combined with [pusing them via the CLI](https://github.com/orgs/supabase/discussions/8165) it would be a much better development flow for email templates

**Comment by hf** [comment]
Support for local testing of email sending has been added after this was asked. I'll close the discussion for now.
